aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -0,0 +1,248 @@
+module AptosFramework::Stake {
+    use Std::Vector;
+    use Std::Signer;
+    use CoreFramework::SystemAddresses;
+    use CoreFramework::Timestamp;
+//    use CoreFramework::Reconfiguration;
+    use AptosFramework::TestCoin::{Self, Coin};
+
+    const MINIMUM_LOCK_PERIOD: u64 = 86400;
+    const MINIMUM_RECONFIG_PERIOD: u64 = 3600;
+
+    /// Basic unit of stake delegation, it's stored in StakePool.
+    struct Delegation has store {
+        coin: Coin,
```

Comment:
@zekun000 - This might be a noob question, but what prevents the validator to spend these delegated coins?

Reply:
the coin is wrapped in the Delegation struct which can only be mutated by the module defines it.